src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -114,6 +117,7 @@ def __init__(
         quantize_embeddings: bool = True,
         reduce_range: bool = False,
         quantize_linear_activations: bool = True,
+        enable_int4_activations: bool = False,
```

Comment:
nit: let's remove `enable_` from the name, since it's implied

Reply:
Done!